src/canmatrix/canmatrix.py

```diff
@@ -323,26 +329,28 @@ def calculateRawRange(self):
         )
 
     def setMin(self, min=None):
+        # type: (canmatrix.types.OptionalPhysicalValue) -> canmatrix.types.OptionalPhysicalValue
```

Comment:
Here the sample usage of type alias which is not particulary useful for mypy (because it is `Any`) but I believe it's useful for human.